hexrd/wppf/WPPF.py

```diff
@@ -307,15 +302,19 @@ def chebyshevfit(self):
             tth = self._tth_list[i]
             self._background.append(Spectrum(x=tth, y=self.init_bkg(tth)))
 
-    # cubic spline fit of background using custom points chosen from plot
-    def splinefit(self, x, y, tth):
+    def splinefit(self, points):
         """
         03/08/2021 SS adding tth as input. this is the
         list of points for which background is estimated
         """
+        self._background = []
+        x = points[:,0]
+        y = points[:,1]
         cs = CubicSpline(x, y)
-        bkg = cs(tth)
-        return Spectrum(x=tth, y=bkg)
+        for i, s in enumerate(self._spectrum_expt):
+            tth = self._tth_list[i]
+            bkg = cs(tth)
+            self._background.append(Spectrum(x=tth, y=cs(tth)))
```

Comment:
```suggestion
            self._background.append(Spectrum(x=tth, y=bkg))
```

Can it be this instead?